rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/InsertCommandQuery.java

```diff
@@ -20,13 +20,14 @@
 
 package org.spine3.server.storage.jdbc.command.query;
 
-import org.spine3.server.storage.CommandStorageRecord;
+import org.spine3.server.command.storage.CommandStorageRecord;
 
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.COMMAND_COL;
```

Comment:
Why don't we migrate to a newer version with `StorageField` and adopt the solution. Or, do you want to do it in a separate PR?